src/plugins/GUITestBase/src/tests/regression_scenarios/GTTestsRegressionScenarios_7001_8000.cpp

```diff
@@ -2787,6 +2788,32 @@ GUI_TEST_CLASS_DEFINITION(test_7611) {
     CHECK_SET_ERR(pdfFileSize > 1000 * 1000, "Invalid PDF file size: " + QString::number(pdfFileSize));
 }
 
+GUI_TEST_CLASS_DEFINITION(test_7621) {
+    // Click "Create Sequence" on the Start Page.
+    // In the "Create Sequence" dialog that appears, open the sequence save dialog.
+    //     Expected: no "QWindowsNativeFileDialogBase::selectNameFilter: Invalid parameter..." message in the log.
+    class OpenCreateSequenceDialog : public CustomScenario {
+    public:
+        void run(GUITestOpStatus& os) override {
+            GTUtilsDialog::waitForDialog(os, new Filler(os, ""));
+            GTLogTracer logTracer("QWindowsNativeFileDialogBase::selectNameFilter: Invalid parameter");
+
+            GTWidget::click(os, GTWidget::findToolButton(os, "browseButton"));
+
+            GTUtilsLog::checkContainsMessage(os, logTracer, false);
+            GTWidget::close(os, GTWidget::getActiveModalWidget(os));
+            GTUtilsDialog::clickButtonBox(os, QDialogButtonBox::Cancel);
+        }
+    };
+    qputenv(ENV_USE_NATIVE_DIALOGS, "1");
```

Comment:
You can't use it in UI tests: different WM/DE may use file dialogs incompatible with our testing system. Even if the test works on Linux the check expression with "QWindowsNativeFileDialogBase" is not meaningful to check at all.

I think we can skip UI test for this bug because tests do not use native file dialogs.

Reply:
I can skip, but note that I did the test only for Windows. Please check that I did it right.

Reply:
I would suggest removing this test completely.
If you want to keep it you can register it for Windows only.

Reply:
I agree with the suggestion, the test has been removed.